packages/components/src/custom-select-control-v2/custom-select.tsx

```diff
@@ -111,7 +111,7 @@ function _CustomSelect(
 					store={ store }
 				/>
 				<Styled.SelectPopover gutter={ 12 } store={ store } sameWidth>
-					<CustomSelectContext.Provider value={ { store } }>
+					<CustomSelectContext.Provider value={ { store, size } }>
```

Comment:
TODO: should probably memoize this object to avoid passing a new instance every time